Geovalidation.md

```diff
@@ -0,0 +1,143 @@
+#Geovalidation API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [Request Structure](#request-structure)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+#Description
+
+Accepts structured or unstructured location information and tests the validity of the resulting resolved geographic coordinate based on the strategies requested and the location types accepted. This service supports the HTTP/GET method only.
+
+----------------------------
+#Request Structure
+
+Example URL: https://api.careerbuilder.com/core/geography/validation?locality=atlanta&country=US&validation_strategies=FREETEXT&accepted_location_types=locality
+
+At least one of (query, locality, postal_code, admin_area, country) must be provided in the query string, as well as a comma separated strings of valid accepted location types, and validation stratgies.
+
+
+| Parameter  | Description |
+|------------|-------------|
+| address      | Optional. A string containing address information, e.g. "5550 Peachtree Pkwy."" Other forms of location data may be passed in this field (such as postal codes or country names), but we recommend using the type-specific parameters when the location type of the data is known. This parameter is handled in the same way as the **query** parameter and the two cannot be used simultaneously. |
```

Comment:
rather than copying and pasting documentation from geocoding (is this really any better than copying and pasting code?) let's change the wording to state that we accept all of the same parameters as the geocoding service, and link to that part of our documentation. we can name the parameters, but let's not duplicate the task of documenting them.